.scrutinizer.yml

```diff
@@ -20,9 +20,11 @@ build:
     dependencies:
         before:
             - "yes '' | pecl -q install -f mongo-stable"
+            - "yes '' | pecl -q install -f mongodb-stable"
             - "sudo composer self-update"
         override:
-            - "composer install --no-dev --no-interaction --prefer-dist"
+            - composer update --no-interaction --prefer-dist
```

Comment:
I'm running coverage in scrutinizer, look at these [lines](https://github.com/Respect/Structural/blob/master/.scrutinizer.yml#L26-L32)